components/HistoricalByDay/HistoricalByDay.tsx

```diff
@@ -0,0 +1,93 @@
+import React, { useEffect, useState } from "react"
+import DatePicker from "react-datepicker"
+import dayjs from "dayjs"
```

Comment:
I'm more familiar with `date-fns` and it is tree-shakeable.  This seems to have a smaller default size, but if we're going to introduce this library prepare to have arguments for why it fits our use cases and is better.